### ISSUE TITLE
Relationship between images_per_batch and  epochSize ?

### ISSUE THREAD

**Issue by gaopeng-eugene** [facebookresearch/multipathnet#26]

**Comment by gaopeng-eugene** [comment]
From my point of view, images_per_batch multiply epochSize should equal to the image numbers? Given one should decide the other. In the original setting, images_per_batch is 2 while epochSize is 200 which means one epoch only look through 400 images?

**Comment by szagoruyko** [comment]
right, epoch size does not depend on the number of images in dataset in this case.
